README.md

```diff
@@ -117,6 +117,7 @@ Metadata has the following attributes:
 * **exception** *Optional* unhandled exception which occurred during scenairo processing.
   * **class** *Required* exception class name.
   * **message** *Optional* exception message.
+* **timestamp** *Optional* ISO 8601 timestamp indicating when the AppMap generation was completed.
```

Comment:
I feel start time would be more useful, especially in conjunction with the `event.timestamp` field. What do you think? Maybe "ISO 8601 timestamp which *should* indicate when the recording started" to keep it open.

Reply:
I guess I will drop it. I don’t have time for this many iterations anyway.